config/directory.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package config
+
+import (
+	"path/filepath"
+	"strconv"
+	"testing"
+)
+
+// TestStepConfigFunc is the callback type used with acceptance tests to
+// specify a string which identifies a directory containing Terraform
+// configuration files.
+type TestStepConfigFunc func(TestStepConfigRequest) string
+
+// TestStepConfigRequest defines the request supplied to types
+// implementing TestStepConfigFunc.
+type TestStepConfigRequest struct {
+	StepNumber int
```

Comment:
I think we may be able to add `TestName string` as well since the "front door" `resource.Test()` and friends should already be accepting a `testing.T`. 👍 I'll put a reciprocal comment on the function as well.

Reply:
Added.